python/ql/src/experimental/semmle/python/Concepts.qll

```diff
@@ -40,5 +42,37 @@ class RegexExecution extends DataFlow::Node {
   RegexExecution() { this = range }
 
   DataFlow::Node getRegexNode() { result = range.getRegexNode() }
+
   Attribute getRegexMethod() { result = range.getRegexMethod() }
```

Comment:
Nitpick: It is probably cleaner to have a `DataFlow::CallCfgNode` here.

Reply:
I meant to return `re.this()` with `getRegexMethod()`, that's why it's set to `Attribute`, but I've misunderstood the class. It'd be so nice to get `re.this()` for the `select` message to show also the method instead of `re.` and the method name, but perhaps it's too much detailed?

Reply:
I can't find a method treating `obj.this` not as an string.

```codeql
  /** Gets the name of this attribute. That is the `name` in `obj.name` */
  string getName() { result = Attribute_.super.getAttr() }
```

Reply:
Well, actually it was doing what I though but now I prefer it to show the entire call (`re.search`) 😆